OpenRA.Mods.Common/Traits/Render/WithSpriteBarrel.cs

```diff
@@ -46,10 +47,10 @@ public IEnumerable<IActorPreview> RenderPreviewSprites(ActorPreviewInitializer i
 			var anim = new Animation(init.World, image, () => t.InitialFacing);
 			anim.Play(RenderSprites.NormalizeSequence(anim, init.GetDamageState(), Sequence));
 
-			var turretOrientation = body.QuantizeOrientation(new WRot(WAngle.Zero, WAngle.Zero, WAngle.FromFacing(t.InitialFacing)), facings);
-			var turretOffset = body.LocalToWorld(t.Offset.Rotate(turretOrientation));
+			Func<WRot> turretOrientation = () => body.QuantizeOrientation(new WRot(WAngle.Zero, WAngle.Zero, WAngle.FromFacing(t.InitialFacing)), facings);
```

Comment:
The turret facing doesn't change, so this can stay as an `int` that you wrap in a `Func` in the ctor below.